testing/btest/core/tunnels/geneve.zeek

```diff
@@ -0,0 +1,13 @@
+# @TEST-EXEC: zeek -b -r $TRACES/tunnels/geneve.pcap %INPUT >out
```

Comment:
The unclear of the pcap is unclear, but I could not find another sample. I added clear attribution to the commit introducing the file, but it is not clear to me whether it can be included.

Worst case we need to drop the patch adding the tests.

Reply:
I'm not sure what to say on this one. How difficult would it be to set up a GENEVE tunnel on a linux box and capture a couple of pings through it?

Reply:
I replaced the PCAP with on from the Wireshark bug tracker which might be marginally better, but still has a somewhat fuzzy license. I'll also ask on Slack whether e.g., @vijitnair can provide a capture file.